specification/cognitiveservices/ContentSafety/routes.tsp

```diff
@@ -67,15 +69,23 @@ interface TextBlocklists {
   @doc("Add or update blocklistItems to a text blocklist. You can add or update at most 100 blocklistItems in one request.")
   addOrUpdateBlocklistItems is BlockOps.ResourceAction<
     TextBlocklist,
-    AddOrUpdateTextBlocklistItemsOptions,
+    {
+      @doc("Options for adding or updating blocklist items.")
+      @projectedName("csharp", "options")
+      options: AddOrUpdateTextBlocklistItemsOptions,
```

Comment:
I learned from Minjue and this change is mainly for change the name of the body parameter not change the body shape. So I think we need to add `@body` annotation here otherwise it would change the body shape to `{ options: xxxx }`.

```
 {
      @doc("Options for adding or updating blocklist items.")
      @projectedName("csharp", "options")
      @body
      body: AddOrUpdateTextBlocklistItemsOptions,
}
```